flink-tests/src/test/scala/org/apache/flink/api/scala/misc/MassiveCaseClassSortingITCase.scala

```diff
@@ -35,11 +35,13 @@ import org.apache.flink.runtime.operators.sort.UnilateralSortMerger
 import org.apache.flink.api.java.typeutils.runtime.RuntimeSerializerFactory
 import org.junit.Assert._
 import org.apache.flink.runtime.jobgraph.tasks.AbstractInvokable
+import org.junit.Test
 
 class MassiveCaseClassSortingITCase {
   
   val SEED : Long = 347569784659278346L
-  
+
+  @Test
```

Comment:
Can you remove this one again? We wrote this as a manual test, because it takes forever to run. Would be great to have a Comment, though, and move it to the package `manual`, like the manual Java tests.